universal-application-tool-0.0.1/app/views/FileUploadViewStrategy.java

```diff
@@ -101,4 +196,14 @@ protected boolean hasUploadedFile(Params params) {
   protected boolean hasAtLeastOneRequiredQuestion(Params params) {
     return params.block().getQuestions().stream().anyMatch(question -> !question.isOptional());
   }
+
+  String acceptFileTypes() {
+    return MIME_TYPES_IMAGES_AND_PDF;
+  }
+
+  ContainerTag errorDiv(Messages messages, FileUploadQuestion fileUploadQuestion) {
```

Comment:
question (blocking): can these two methods be `protected`?

Reply:
Fixed.